dev.sh

```diff
@@ -118,4 +118,4 @@ echo
 sleep 10
 
 yarn dev:secrets:gen
-concurrently "yarn run start" "yarn run compose:deps"
\ No newline at end of file
+concurrently "yarn run start" "yarn run compose:deps" "bash setup-elastic-index.sh"
```

Comment:
We should add this for the case when we are running only the dependencies with `--only-dependencies`

Reply:
yes. I almost forgot. Thanks.